.github/workflows/publish.yml

```diff
@@ -10,16 +10,16 @@ permissions:
 
 jobs:
   publish_maplibre_gl_platform_interface:
-    uses: ./.github/workflows/publish_single.yml
+    uses: .github/workflows/publish-single.yml@main
```

Comment:
And no branch. See https://stackoverflow.com/a/76614274

Reply:
fixed, used your solution in ceb8ba86d2067fe97628f5f25ca2f948c03fbb4a